src/js/languages/default.json

```diff
@@ -94,6 +94,22 @@
     "multiple": "multiple",
     "selected": "; Selected: {currentSelectedValue}"
   },
+  "selectMultiple": {
+    "multiple": "{selectedCount} of {totalCount} selected",
```

Comment:
```suggestion
    "summarizedValue": "{selected} of {total} selected",
```

Reply:
These suggestions with changes for the value will also require changes in the component files to make sure the variable section is passed through properly.